### PR TITLE
Linux prerequisites

### PR DESCRIPTION
**Pull request by Alecaddd**

### CODE REVIEW

File: the-basics/building-thunderbird/linux-build-prerequisites.md [hunk 1]
```diff
@@ -6,3 +6,58 @@ description: >-
...
+The Thunderbird build will take up to 30 GB of disk space in order to complete. Be sure to have enough free space and a fast internet connection to avoid interruptions.
+
+## Bootstrap your system
+
+Access the location where you downloaded the `mozilla-central` source code, most likely `source/` and trigger this command:
+
+```text
+./mach bootstrap
+```
+
```
Comment (ryanleesipes) [inline the-basics/building-thunderbird/linux-build-prerequisites.md]: Should we put something here that tells the user what to select from this list?

Please choose the version of Firefox you want to build:
1. Firefox for Desktop Artifact Mode
2. Firefox for Desktop
3. GeckoView/Firefox for Android Artifact Mode
4. GeckoView/Firefox for Android

I selected number 2. I assume that is the only one that works?

Comment (Alecaddd) [reply the-basics/building-thunderbird/linux-build-prerequisites.md]: Mh, I don't think it's necessary as those choices are thoroughly explained in the terminal once the `bootstrap` command is running.
I personally selected option 1 to have those C artifacts already built and speed up my first building process.

Comment (ryanleesipes) [reply the-basics/building-thunderbird/linux-build-prerequisites.md]: I'll pull it in and we can think on it more.

Comment (mkmelin) [reply the-basics/building-thunderbird/linux-build-prerequisites.md]: Artifact Mode doesn't work for Thunderbird (though Geoff is working on fixing that).

**Pull request merged**